WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/FeeLineDetailsViewModelTests.swift

```diff
@@ -302,4 +301,34 @@ final class FeeLineDetailsViewModelTests: XCTestCase {
         // Then
         XCTAssertEqual(savedFeeLine?.taxStatus, .taxable)
     }
+
+    func test_view_model_amount_placeholder_has_expected_value() {
+        // Given
+        let viewModel = FeeLineDetailsViewModel(isExistingFeeLine: false,
+                                                baseAmountForPercentage: 0,
+                                                feesTotal: "",
+                                                locale: usLocale,
+                                                storeCurrencySettings: usStoreSettings,
+                                                didSelectSave: { _ in })
+
+        // Then
+        XCTAssertEqual(viewModel.amountPlaceholder, "0")
+    }
+
+    func test_view_model_initializes_correctly_with_no_existing_fee_line() {
+        // Given
+        let viewModel = FeeLineDetailsViewModel(isExistingFeeLine: false,
+                                                baseAmountForPercentage: 0,
+                                                feesTotal: "",
+                                                locale: usLocale,
+                                                storeCurrencySettings: usStoreSettings,
+                                                didSelectSave: { _ in })
+
+        // When
+        viewModel.amount = "hi:11.3005.02-"
```

Comment:
I don't think we need amount input in this test. Just as previous one, it only checks the state after initialisation, so we can remove "When" section altogether.